autogpt_platform/frontend/src/app/login/page.tsx

```diff
@@ -14,40 +12,68 @@ import { useForm } from "react-hook-form";
 import { Input } from "@/components/ui/input";
 import { z } from "zod";
 import { zodResolver } from "@hookform/resolvers/zod";
-import { PasswordInput } from "@/components/PasswordInput";
-import { FaGoogle, FaGithub, FaDiscord, FaSpinner } from "react-icons/fa";
-import { useState } from "react";
+import { useCallback, useState } from "react";
 import { useRouter } from "next/navigation";
 import Link from "next/link";
-import { Checkbox } from "@/components/ui/checkbox";
 import useSupabase from "@/hooks/useSupabase";
 import Spinner from "@/components/Spinner";
-import { useBackendAPI } from "@/lib/autogpt-server-api/context";
-
-const loginFormSchema = z.object({
-  email: z.string().email().min(2).max(64),
-  password: z.string().min(6).max(64),
-  agreeToTerms: z.boolean().refine((value) => value === true, {
-    message: "You must agree to the Terms of Use and Privacy Policy",
-  }),
-});
+import {
+  AuthCard,
+  AuthHeader,
+  AuthButton,
+  AuthFeedback,
+  AuthBottomText,
+  PasswordInput,
+} from "@/components/auth";
+import { loginFormSchema } from "@/types/auth";
 
 export default function LoginPage() {
   const { supabase, user, isUserLoading } = useSupabase();
   const [feedback, setFeedback] = useState<string | null>(null);
   const router = useRouter();
   const [isLoading, setIsLoading] = useState(false);
-  const api = useBackendAPI();
 
   const form = useForm<z.infer<typeof loginFormSchema>>({
     resolver: zodResolver(loginFormSchema),
     defaultValues: {
       email: "",
       password: "",
-      agreeToTerms: false,
     },
   });
 
+  // const onProviderLogin = useCallback(async (
```

Comment:
Remove this?

Reply:
This is valid code that should be uncommented when we enable social login, I added a `TODO`.